en/developer_guide/bundle_configuration.rst

```diff
@@ -0,0 +1,178 @@
++===================+=====================+=====================================================+
+| workflow_function | [class, repository] | The workflow function document and repository class |
++-------------------+---------------------+-----------------------------------------------------+
+| workflow_right    | [class, repository] | The workflow right document and repository class    |
++-------------------+---------------------+-----------------------------------------------------+
+| authorization     | [class]             | The authorization document and repository class     |
++-------------------+---------------------+-----------------------------------------------------+
+| reference         | [class]             | The reference document and repository class         |
++-------------------+---------------------+-----------------------------------------------------+
+
+BackofficeBundle
+~~~~~~~~~~~~~~~~
+
++------------------+-----------------+---------------------------------------------------------------------+
+| Key              | Default value   | Usage                                                               |
++==================+=================+=====================================================================+
+| front_languages  | [$key => $name] | Language in which the front is available, $key is the language code |
++------------------+-----------------+---------------------------------------------------------------------+
+| blocks           | []              | All new blocks you add                                              |
++------------------+-----------------+---------------------------------------------------------------------+
+| fixed_attributes | []              | All the fixed attributes in the blocks                              |
++------------------+-----------------+---------------------------------------------------------------------+
+| field_types      | []              | All the field type available for the content type attributes        |
++------------------+-----------------+---------------------------------------------------------------------+
+| options          | []              | All the options that can be added to the content type attributes    |
++------------------+-----------------+---------------------------------------------------------------------+
+| available_color  | [$key => $name] | Colors available in the backoffice (for the status for instance)    |
++------------------+-----------------+---------------------------------------------------------------------+
+
+LogBundle
+~~~~~~~~~
+
++-----+---------------------+---------------------------------------+
+| Key | Default value       | Usage                                 |
++=====+=====================+=======================================+
+| log | [class, repository] | The log document and repository class |
++-----+---------------------+---------------------------------------+
```

Comment:
you forgot frontBundle 
->arrayNode('devices')
    ->useAttributeAsKey('name')
    ->prototype('array')
        ->children()
            ->scalarNode('parent')->defaultNull()->end()
        ->end()
->end()